docker-compose.yml

```
@@ -1,24 +1,109 @@
+version: "3.9"
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)